google/resource_compute_target_pool_test.go

```diff
@@ -73,6 +77,27 @@ func testAccCheckComputeTargetPoolExists(n string) resource.TestCheckFunc {
 	}
 }
 
+func testAccCheckComputeTargetPoolHealthCheck(targetPool, healthCheck string) resource.TestCheckFunc {
+	return func(s *terraform.State) error {
+		targetPoolRes, ok := s.RootModule().Resources[targetPool]
+		if !ok {
+			return fmt.Errorf("Not found: %s", targetPool)
+		}
+
+		healthCheckRes, ok := s.RootModule().Resources[healthCheck]
+		if !ok {
+			return fmt.Errorf("Not found: %s", healthCheck)
+		}
+
+		hcLink := healthCheckRes.Primary.Attributes["self_link"]
+		if targetPoolRes.Primary.Attributes["health_checks.0"] != hcLink {
+			return fmt.Errorf("Health check not set up. Expected '%s'", hcLink)
```

Comment:
fyi: '%s' is equivalent to %q (up to you if you want to change it)

Reply:
Will do